core/java/android/provider/Settings.java

```diff
@@ -16024,6 +16033,16 @@ public static final class Global extends NameValueTable {
                 CLOCKWORK_HOME_READY,
         };
 
+        /**
+         * Global settings that are moved to non-persisted state.
+         *
+         * @hide
+         */
+        public static final String[] MOVED_TO_TRANSIENT_SETTINGS = {
+                ADD_USERS_WHEN_LOCKED,
+                ENABLE_EPHEMERAL_FEATURE,
```

Comment:
I think a better solution would be to remove definitions of all these keys.
It'll ensure that all places that read them are dealt with.